themes/default/content/registry/packages/threefold/_index.md

```diff
@@ -5,3 +5,91 @@ layout: package
 ---
 
 The Threefold Resource Provider for the [threefold grid](https://threefold.io) lets you manage your infrastructure using pulumi.
+
+## Example
+
+{{< chooser language "go,yml" >}}
```

Comment:
@cnunciato @sean1588 does the language chooser support both `yaml` and `yml`?

Reply:
No, it only supports `yaml`.

Reply:
@rawdaGastan we fixed your PR here for you: `yml` should be `yaml` on line 11, 56 & 58 of `_index.md`. Please apply this change to that file in your upstream provider repository.

Reply:
This ought to do it: https://github.com/threefoldtech/pulumi-threefold/pull/96